autoprecompiles/src/lib.rs

```diff
@@ -286,7 +287,10 @@ impl<'a, M, B: Clone, C: Clone> Clone for VmConfig<'a, M, B, C> {
     }
 }
 
-pub trait InstructionHandler<T, I> {
+pub trait InstructionHandler<T, I, D> {
+    /// Returns the AIR id for the given instruction.
+    fn get_instruction_air_id(&self, instruction: &I) -> D;
+
     /// Returns the AIR for the given instruction.
     fn get_instruction_air(&self, instruction: &I) -> &SymbolicMachine<T>;
```

Comment:
Can these be a single method? Would it make sense?

Reply:
My argument would be that they either both succeed, or both fail, so having a single method would encode that.

Reply:
Right but that's fine, because it's done on the apc crate side. The goal is to simplify client use.